jsonnet/thanos-querier.jsonnet

```diff
@@ -156,6 +156,7 @@ local authorizationRole =
         service.mixin.spec.withPorts([
           ports.newNamed('web', 9091, 'web'),
           ports.newNamed('tenancy', 9092, 'tenancy'),
+          ports.newNamed('http', 9090, 'http'),
```

Comment:
This should be removed once we have ThanosRuler configured with TLS to access the `web` port.  Same with the change to `--http-address` and the additional `ports` config.  But I guess we can leave this for now as a workaround?

Reply:
Will remove, we should have the TLS working now.